src/main/webapp/WEB-INF/views/user/my-bookings.jsp

```diff
@@ -66,11 +66,14 @@
 
     </div>
 </div>
-<c:if test="${pageContext.response.locale=='ua'}">
-    <script src="resources/js/validation-my-booking-ua.js"></script>
-</c:if>
-<c:if test="${pageContext.response.locale != 'ua'}">
-    <script src="resources/js/validation-my-booking.js"></script>
-</c:if>
+ <script src="resources/js/validation-my-booking.js"></script>
+ <c:choose>
+<c:when test="${pageContext.response.locale =='ua'}">
+    <script src="resources/js/lib/messages-ua.js"></script>
+</c:when>
 
+<c:when test="${pageContext.response.locale != 'ua'}">
+    <script src="resources/js/lib/messages.js"></script>
+</c:when>
+</c:choose>
```

Comment:
Your code is good. I like this idea.. But I have a question)
It's looks like Fabric design pattern. The intent of this design pattern is that we have a common interface and different implementations of it. And in runtime user (or program etc) selects what implementation he wants...
So, Can we create another jsp file, where we can implement different situations of choosing languages (like you do here) and then simply append(import) only one file here? Because you use here and in file register-kid.jsp similar code.. If we will have more languages (or more files) it will be the problem of more duplicate code..